cmd/harborAPI.go

```diff
@@ -170,13 +212,89 @@ func Trigger(shipment string, env string) {
 
 	_, body, err := gorequest.New().
 		Post(uri).
-		End()
+		EndBytes()
 
 	if err != nil {
 		log.Fatal(err)
 	}
 
+	var response TriggerResponse
+	unmarshalErr := json.Unmarshal(body, &response)
+	if unmarshalErr != nil {
+		log.Fatal(unmarshalErr)
+	}
+
 	if Verbose {
-		log.Println(body)
+		log.Println(response)
+	}
+
+	return response
+}
+
+// SaveEnvVar saves envvars by doing a delete/add against the api
+func SaveEnvVar(token string, shipment string, composeShipment ComposeShipment, envVarPayload EnvVarPayload, container string) {
+
+	templateString := shipItURI + "/v1/shipment/{shipment}/environment/{env}/envvar/{envvar}"
+
+	//build url
+	//DELETE /v1/shipment/%s/environment/%s/envVar
+	values := make(map[string]interface{})
+	values["shipment"] = shipment
+	values["env"] = composeShipment.Env
+	values["envvar"] = envVarPayload.Name
+
+	if len(container) > 0 {
+		values["container"] = container
+		templateString = shipItURI + "/v1/shipment/{shipment}/environment/{env}/container/{container}/envvar/{envvar}"
+	}
+
+	template, _ := uritemplates.Parse(templateString)
+	url, _ := template.Expand(values)
+
+	//issue delete call
+	//api will return 422 if the envvar doesn't exist, which can be ignored
+	res, _, _ := delete(token, url)
```

Comment:
I thought I had tried those a while back and had issues.  Let's just defer this for now until we rewrite up.  I just want to get this early version out the door.